### PR TITLE
Feature/cc 0125 deposit display

### PR DESCRIPTION
**Pull request by GLC-coder**
feature/cc-0125-deposit-display is ready to be  merged into main
Screen Shot 2022-11-22 at 3 39 26 pm
Screen Shot 2022-11-22 at 3 39 39 pm

### CODE REVIEW

File: src/components/ShoppingCart/CartListItem.tsx [hunk 1]
```diff
@@ -78,10 +81,12 @@ const CartListItem = ({
           </Box>
         </Td>
         <Td verticalAlign="top">
-          <Text style={columnStyle}>A${quotation}</Text>
+          <Text style={columnStyle}>{formatCurrency(quotation)}</Text>
         </Td>
         <Td verticalAlign="top">
-          <Text style={columnStyle}>A${(parseFloat(quotation) * 0.02).toFixed(2)}</Text>
+          <Text style={columnStyle}>
+            {formatCurrency((Number(quotation) * depositRate).toString())}
```
Comment (Double-Twelve) [inline src/components/ShoppingCart/CartListItem.tsx:88]: do not need _toString()_

File: src/components/PriceBar/BudgetBoard.tsx [hunk 2]
```diff
@@ -64,32 +68,37 @@ const BudgetBoard = ({ useTotalPrice }: IBudgetBoardprops) => {
...
-        <Text fontSize={{ base: "xs", lg: "sm" }} fontWeight="600" marginLeft="50px">
-          Estimated Budget
-        </Text>
-        <Text
-          fontSize={{ base: "xs", lg: "sm" }}
-          fontWeight={{ base: "700", lg: "800" }}
-          marginLeft="8px"
+      <Flex flexDirection="column" justifyContent="center" width="40%" color="fontcolor.lightDark">
+        <Flex justifyContent="space-around" fontSize="12px">
+          {" "}
```
Comment (Double-Twelve) [inline src/components/PriceBar/BudgetBoard.tsx:79]: please remove empty space

File: src/components/PriceBar/BudgetBoard.tsx [hunk 3]
```diff
@@ -64,32 +68,37 @@ const BudgetBoard = ({ useTotalPrice }: IBudgetBoardprops) => {
...
-          fontSize={{ base: "md", lg: "lg" }}
-          fontWeight={{ base: "700", lg: "800" }}
-          marginLeft="5px"
-          marginRight="20px"
-        >
-          ${useTotalPrice === "0.00" ? "Loading..." : useTotalPrice}
-        </Text>
-      </Center>
+          <Center>
+            <BsQuestionCircle />
```
Comment (Double-Twelve) [inline src/components/PriceBar/BudgetBoard.tsx:91]: why need to add this icon?

Comment (GLC-coder) [reply src/components/PriceBar/BudgetBoard.tsx:91]: well, I saw it was on the Figma UI.
Screen Shot 2022-11-23 at 11 19 50 pm

**Review by STY1997 — APPROVED** [review]
lgtm

**Review by Double-Twelve — APPROVED** [review]

**Pull request merged**